omod-2.2/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_2/PasswordResetController2_2Test.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web.v1_0.controller.openmrs2_2;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.openmrs.Person;
+import org.openmrs.PersonName;
+import org.openmrs.User;
+import org.openmrs.api.UserService;
+import org.openmrs.api.context.Context;
+import org.openmrs.api.db.LoginCredential;
+import org.openmrs.api.db.UserDAO;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.RestControllerTestUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+public class PasswordResetController2_2Test extends RestControllerTestUtils {
+	
+	private static final String RESET_PASSWORD_URI = "passwordreset";
+	
+	@Rule
+	public ExpectedException expectedException = ExpectedException.none();
+	
+	@Autowired
+	@Qualifier("userService")
+	private UserService userService;
+	
+	@Autowired
+	private UserDAO dao;
+	
+	@Before
+	public void setup() {
+		userService = Context.getUserService();
+	}
+	
+	@Test
+	public void requestPasswordReset_shouldCreateUserActivationKeyGivenUsername() throws Exception {
+		User u = new User();
```

Comment:
Why don't you use an existing user?

Reply:
@wluyima I have been checking and the last dataset that contained an existing user was defined since 1.8 so i am thinking declaring a new user here is appropriate.